cmake/Platform/Linux/Packaging/snapcraft.yaml.in

```
@@ -0,0 +1,66 @@
+name: o3de
+version: '${O3DE_INSTALL_VERSION_STRING}'
+summary: O3DE Engine
+description: |
+  Open 3D Engine (O3DE) is an Apache 2.0-licensed multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations. 
+license: Apache-2.0
+confinement: classic
+base: core22
+
+parts:
+  test-o3de:
```

This is probably arbitrary but maybe rename this to `o3de` not `test-o3de` if it is OK to have a part with the same name as the snap
```suggestion
  o3de:
```

---

Yes, that's allowed, however this was also something missed during renaming. Fixed so it makes sense :)